tests/test_autograd_hanging_var.py

```diff
@@ -0,0 +1,11 @@
+import minpy.numpy as np
+from minpy.core import grad
+
+
+def func(x):
+	y = x + 1
+	return x
+
+x = 1
+gradient = grad(func)
+print gradient(x)
```

Comment:
Plz use `from __future__ import print_function` and write code that's compatible with both Python 2 and 3